src/pages/css/css-pre-processors/index.md

```diff
@@ -3,7 +3,8 @@ title: CSS Pre-Processors
 ---
 ## CSS Pre-Processors
 
-Pre-processor is a software that allows to use extended CSS functionality with variables, functions, operators and many more useful assets that plain CSS is missing.
+Pre-processor is a software that allows to use extended CSS functionality with variables, functions, operators and many more useful assets that plain CSS is missing.Raw CSS is powerful but is not viable in the longer run, in organizations where code 
+exceeds 10kloc it is not viable to use vanilla CSS. CSS pre-processers enhance the power of CSS making it viable and less cumbersome for large projects.
```

Comment:
Hi @geekyatri 

Thank you for opening this PR. Currently the addition you are proposing to the page isn't substantial. It's turning the page into a wall of text that doesn't really explain the function of CSS Pre-Processors.

Can you edit this and instead list features that CSS Pre-Processors provide? Such as splitting of files, importing, mixins, variables and examples of some of these features?

Reply:
Will do